volunteers.html

```diff
@@ -61,147 +63,377 @@
 		<div class="container team">
 
 		  <div class="row">
-        <h1>Instructors</h1>
+        <h1>Volunteers -- you made this happen!</h1>
```

Comment:
If we are trying to make this page sensible without javascript, then adding people to the bottom of the page is the way to go.  Maybe we'll add sorting after the fact as a 'nice feature to have' but not something to build right now.